### DISCUSSION TITLE
Sidkick profile selector dont work?

### DISCUSSION THREAD

**Discussion by ChrisNL74** [frntc/SIDKick#1] [Q&A]
Hi guys, 
Im new here....

I have a sidkick installed but have some troubles with the profile selector. The issue is that i dont here a different by changing the settings of the 6581 or 8580 sid sound. Sam speech is working but no changing.

My sidkick is installed in a sixtyclone c64 board and im using a u2+.
I dont know what the problem should be can someone help me?

Thanks,
Kind regards,
Chris.

**Comment by frntc** [comment]
Hi, 

I assume the SIDKick works apart from the profile selector, so I'd factor the Sixtyclone and U2 out. Do the profiles sound different, when you activate them manuall in the config tool? (maybe you'd want to define very differently sounding profiles for testing, e.g. different volumes/pannings)

**Comment by ChrisNL74** [comment]
Hi frantic It does something but I can’t setup as example Tetris. The arp is really error.How can I setup this with the right settings?Verstuurd vanaf mijn iPhoneOp 20 sep 2023 om 21:51 heeft frntc ***@***.***> het volgende geschreven:﻿
Hi,
I assume the SIDKick works apart from the profile selector, so I'd factor the Sixtyclone and U2 out. Do the profiles sound different, when you activate them manuall in the config tool? (maybe you'd want to define very differently sounding profiles for testing, e.g. different volumes/pannings)

**Reply by frntc** [reply]
sorry, not sure I get what you mean!?

**Comment by ChrisNL74** [comment]
The selector is working but the sound is not really good I can’t adjust it. Try as example: Wally Beben - Tetris.sid.Verstuurd vanaf mijn iPhoneOp 26 sep 2023 om 17:25 heeft frntc ***@***.***> het volgende geschreven:﻿
sorry, not sure I get what you mean!?

**Comment by ChrisNL74** [comment]
Hi, 

I admire youre sidkick pico especially the musical led.

Where can I order that led or what is the component name?

Like to hear,

Kind regards,
Chris.




Verstuurd vanaf mijn iPhone

**Comment by marty41** [comment]
Are you refering to the WS2812B LED? Just have a look into the great BOM Frenetic offers. ;) 
https://github.com/frntc/SIDKick-pico/blob/main/BOM/ibom_SKpico2040DAC.html

**Reply by frntc** [reply]
well, this is the other SKpico project, but the LEDs are the same ;-)

@ChrisNL74 these LEDs are also available in stripes from various resellers
